paladin-core/src/runtime/mod.rs

```diff
@@ -122,8 +122,9 @@ pub struct Runtime {
     _marker: Marker,
 }
 
-const IPC_ROUTING_KEY: &str = "ipc-routing-key";
-pub const DEFAULT_ROUTING_KEY: &str = "default";
```

Comment:
this is used in zk evm
we need to make a change there as well

Reply:
Done here https://github.com/0xPolygonZero/zk_evm/pull/748